packages/shape-to-query/package.json

```diff
@@ -2,7 +2,8 @@
   "name": "@hydrofoil/shape-to-query",
   "version": "0.3.2",
   "scripts": {
-    "prepack": "tsc"
+    "prepack": "tsc",
+    "prepare": "npm run prepack"
```

Comment:
Curious, why did you add `prepare`?

Reply:
This is just so I can locally install it for what I'm working on at the moment; will remove when I take this out of draft.

Reply:
gotcha